fastjet/internal/plot/delaunay_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plot
+
+import (
+	"bytes"
+	"image"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"go-hep.org/x/hep/fastjet/internal/delaunay"
+)
+
+func TestDelaunayPlot(t *testing.T) {
+	fname := "delaunay_example.png"
+	ref := "testdata/delaunay_example.png"
+	err := DelaunayPlotExample(fname)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want, err := ioutil.ReadFile(ref)
```

Comment:
perhaps the whole `ioutil.ReadFile(name)` + `image.Decode(...)` code stanza could be refactored into a single inlined function:

```go
read := func(name string) image.Image {
    raw, err := ioutil.ReadFile(name)
    if err != nil {
        t.Fatalf("error reading %s: %v", name, err)
    }

    img, _, err := image.Decode(bytes.NewReader(raw))
    if err != nil {
        t.Fatalf("error decoding %s: %v", name, err)
    }
    return img
}

got := read(fname)
want := read(ref)
```

Reply:
Done

I have never used inline functions in go. It looks pretty convenient.